### DISCUSSION TITLE
aspnetcore-in-action-2e-applying-the-mvc-design-pattern-to-razor-pages/

### DISCUSSION THREAD

**Discussion by giscus** [andrewlock/blog-comments#428] [Announcements]
# aspnetcore-in-action-2e-applying-the-mvc-design-pattern-to-razor-pages/

In this article from my new book we look in greater depth at how the MVC design pattern applies to Razor Pages in ASP.NET Core.

https://andrewlock.net/aspnetcore-in-action-2e-applying-the-mvc-design-pattern-to-razor-pages/

**Comment by MikeOtown** [comment]
complete working project code?
